src/gazebo_gimbal_controller_plugin.cpp

```diff
@@ -348,7 +348,22 @@ void GimbalControllerPlugin::Load(physics::ModelPtr _model,
     gzerr << "GimbalControllerPlugin::Load ERROR! Can't get imu sensor '"
           << cameraImuSensorName << "' " << endl;
   }
+  
+  const char *host_ip = std::getenv("PX4_VIDEO_HOST_IP");
+  if (host_ip) {
+    this->udp_gimbal_host_ip = std::string(host_ip);
+  } else if (this->sdf->HasElement("udp_gimbal_host_ip")) {
+    this->udp_gimbal_host_ip =  _sdf->Get<std::string>("udp_gimbal_host_ip");
+  } else {
+    this->udp_gimbal_host_ip = "127.0.0.1";
+  }
 
+  if (this->sdf->HasElement("udp_gimbal_port_remote")) {
+    this->udp_gimbal_port_remote = _sdf->Get<int>("udp_gimbal_port_remote");
+  } else {
+    this->udp_gimbal_port_remote = 13030;
+  }
+  gzwarn << "[gazebo_gimbal_controller_plugin] Streaming gimbal mavlink stream to ip: " << this->udp_gimbal_host_ip  << " port: " << this->udp_gimbal_port_remote << std::endl;
```

Comment:
Should that be a warning or just info?

Reply:
to me it makes sense for it to be an info, but I just wanted to mimic the way it has been done there : 
https://github.com/PX4/PX4-SITL_gazebo/blob/6c3f1f8af6d062e4225a152a6d2a7bc5e276f537/src/gazebo_gst_camera_plugin.cpp#L232